### ISSUE TITLE
Instructions

### ISSUE THREAD

**Issue by kththomas** [tartanguru/alexa-assistant#19]
Hi, Just a small point on the instructions.
Under the section "Enable Google Assistant API:-" point 19 
You need to add  "press Save" before the comment "You can now close this tab/window"

Regards,

Keith Thomas

**Comment by okko** [comment]
The Client ID should already be created in step 17, while in steps 18 and 19 its credentials are only viewed, so no saving needed in step 19 before closing?

**Comment by tartanguru** [comment]
I'm sorry but I'm not understanding what the issue is?
The client ID is created in step 17, but you need to undertake steps 18 and
19 to get the client secret.
There is no mention of the need to save but there is no harm in doing so

**Comment by kththomas** [comment]
Thanks for checking this.  I'll describe what happened, I finished the "Enable Google Assistant API" section, noted the Client ID and client secret and ended the session without saving.  When I continued the setup the next day I could not find the Google project I had created.  I went through the steps again and this time pressed save at the end and this seemed to correct the problem. If you think this should be unnecessary then I probably made some mistake on the first attempt and lost the setup.
Thanks once again for investigating this.
